### ISSUE TITLE
can't execute nativeMissingDependencies task in parallel

### ISSUE THREAD

**Issue by xuwei-k** [scala-native/scala-native#532]
https://github.com/xuwei-k/nativeMissingDependencies-parallel

```
java.nio.file.FileSystemAlreadyExistsException
	at com.sun.nio.zipfs.ZipFileSystemProvider.newFileSystem(ZipFileSystemProvider.java:113)
	at java.nio.file.FileSystems.newFileSystem(FileSystems.java:326)
	at java.nio.file.FileSystems.newFileSystem(FileSystems.java:276)
	at scala.scalanative.io.VirtualDirectory$JarDirectory.<init>(VirtualDirectory.scala:151)
	at scala.scalanative.io.VirtualDirectory$.jar(VirtualDirectory.scala:57)
	at scala.scalanative.io.VirtualDirectory$.real(VirtualDirectory.scala:65)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$availableDependenciesTask$1$$anonfun$apply$4$$anonfun$17.apply(ScalaNativePluginInternal.scala:255)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$availableDependenciesTask$1$$anonfun$apply$4$$anonfun$17.apply(ScalaNativePluginInternal.scala:254)
	at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:251)
	at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:251)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.flatMap(TraversableLike.scala:251)
	at scala.collection.AbstractTraversable.flatMap(Traversable.scala:105)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$availableDependenciesTask$1$$anonfun$apply$4.apply(ScalaNativePluginInternal.scala:254)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$availableDependenciesTask$1$$anonfun$apply$4.apply(ScalaNativePluginInternal.scala:251)
	at scala.scalanative.util.Scope$.apply(Scope.scala:30)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$availableDependenciesTask$1.apply(ScalaNativePluginInternal.scala:251)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$availableDependenciesTask$1.apply(ScalaNativePluginInternal.scala:251)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by densh** [comment]
Thanks for the bug report!

**Comment by densh** [comment]
Fixed by #1022
